core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.spec.ts

```diff
@@ -1000,6 +1089,15 @@ describe('EditEmaStore', () => {
                 });
             });
 
+            it('should call unlock page service', () => {
+                const dotContentletLockerService = spectator.inject(DotContentletLockerService);
+                const spyUnlock = jest.spyOn(dotContentletLockerService, 'unlock');
+
+                spectator.service.unlockPage('123');
+
+                expect(spyUnlock).toHaveBeenCalledWith('123');
+            });
```

Comment:
This updater patch the state and you're not checking that change.

Reply:
Fixed!